WIP: Certifications learning (#102)
madetech/learn: certifications/README.md

> # Certifications

Reviewer: ```suggestion
# AWS Certification
```

Reviewer: Would also move to `certification/aws/README.md`

> * [Sign up for the test](https://aws.amazon.com/certification/)

Reviewer: ```suggestion
To sign for the exam go to [aws.amazon.com/certification](https://aws.amazon.com/certification/)
```

> * [REALLY good resource](https://acloud.guru/learn/)

Reviewer: ```suggestion

* [Introduction to Cloud Computing](https://acloud.guru/learn/) by A Cloud Guru.
```
